src/v/cluster/scheduling/allocation_node.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "cluster/scheduling/types.h"
+#include "cluster/types.h"
+#include "model/fundamental.h"
+
+#include <absl/container/node_hash_map.h>
+
+namespace cluster {
+
+class allocation_state;
+/**
+ * Allocation node represent a node where partitions may be allocated
+ */
+class allocation_node {
+public:
+    static constexpr const uint32_t core0_extra_weight = 2;
+    // TODO make configurable
+    static constexpr const uint32_t max_allocations_per_core = 7000;
+
+    allocation_node(
```

Comment:
can you give names here to the parameters. what's `uint32_t` ? `named_type<>`'s are easy to read, but the others aren't.

Reply:
done

Reply:
i don't see it ... did you forget to push?